.github/workflows/python-app.yml

```diff
@@ -0,0 +1,36 @@
+# This workflow will install Python dependencies, run tests and black with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
```

Comment:
based on our travis environment, pycytominer only supports python 3.7, 3.8 and 3.9 https://github.com/cytomining/pycytominer/pull/151/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485L3-L5

can you align the versions here?